src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/CngSymmetricAlgorithmCore.cs

```diff
@@ -162,8 +182,6 @@ private UniversalCryptoTransform CreateCryptoTransform(byte[] rgbKey!!, byte[]?
             // is correct, and detached from the input parameter.
             byte[]? iv = mode.GetCipherIv(rgbIV).CloneByteArray();
```

Comment:
The IV validation between CreateTransform and CreateLiteCipher are pretty matched, except CreateLiteCipher has a missing-IV check.  Does that happen elsewhere for CreateTransform?

Reply:
Yeah that happens in `Helpers.GetCipherIv` (an extension method on `CipherMode`...), but that is built on byte arrays, so I replicated the null check to use Span in the Lite path.